canonical-web-and-design/charmhub.io
# Remove libraries introduction

## Done
- Remove libraries introduction

## How to QA
- Check a charm with libraries like https://charmhub-io-1341.demos.haus/nginx-ingress-integrator/libraries
- Check a charm without libraries https://charmhub-io-1341.demos.haus/graylog-k8s/libraries

## Issue / Card
Fixes #1332
